### ISSUE TITLE
bug: missing distribution types

### ISSUE THREAD

**Issue by ajtorres9** [jeffreylanters/react-unity-webgl#512]
### Please avoid duplicates

- [X] I checked [all open issues](https://github.com/jeffreylanters/react-unity-webgl/issues) and [discussions](https://github.com/jeffreylanters/react-unity-webgl/discussions) and none of them matched my problem.

### Language and Compiler

Other TypeScript compiler

### What environment are you using?

Local Development Server

### When does your problem occur?

While writing code

### What does your problem relate to?

The problem seems Module related

### React-Unity-WebGL Version

9.4.1

### React Version

18.2.0

### Unity Version

2022.3.0f1

### What happened?

The latest release appears to be missing the `distribution/types` folder which produces broken types within a TypeScript application.

Version 9.4.1 on NPM: https://www.npmjs.com/package/react-unity-webgl?activeTab=code

### Example:
![Screenshot 2023-05-31 at 10 46 14 AM](https://github.com/jeffreylanters/react-unity-webgl/assets/450495/94e8401e-8674-42e4-9304-70261942525d)

![Screenshot 2023-05-31 at 10 47 51 AM](https://github.com/jeffreylanters/react-unity-webgl/assets/450495/f601cb5c-a3f0-4574-a779-98fbbfce045d)

### Reproducible test case

```tsx
import { useUnityContext } from 'react-unity-webgl'

export const Example = () => {
  const { addEventListener } = useUnityContext({
    codeUrl: `/build/code.wasm.br`,
    dataUrl: `/build/asset.data.br`,
    frameworkUrl: `/build/framework.js.br`,
    loaderUrl: `/build/loader.js`,
    streamingAssetsUrl: `/streaming-assets`,
  })

  return null
}
```

### Would you be interested in contributing a fix?

- [ ] yes, I would like to contribute a fix

**Comment by jeffreylanters** [comment]
Thanks for the report. You are right! I'm going to look into it!

**Comment by jeffreylanters** [comment]
Sorry for the delay. The issue is resolved in [the latest release v9.4.2](https://github.com/jeffreylanters/react-unity-webgl/releases/tag/v9.4.2).
